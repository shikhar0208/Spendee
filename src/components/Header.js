import React from 'react';
import {NavLink} from "react-router-dom";

const Header = () => (
    <header>
        <h1>Spendee</h1>
        <NavLink to="/" activeClassName="is-active" exact={true}>Dashboard</NavLink>
        <NavLink to="/Create" activeClassName="is-active" >Create Expense</NavLink>
    </header>
);

export default Header