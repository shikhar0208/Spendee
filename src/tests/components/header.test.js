import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

test('should render hesder correctly', () => {
    const wrapper = shallow(<Header />)
    expect(wrapper).toMatchSnapshot();
    // expect(wrapper.find('h1').text()).toBe('Spendee');
    // const renderer = new ReactShallowRenderer();
    // renderer.render(<Header />)
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
});