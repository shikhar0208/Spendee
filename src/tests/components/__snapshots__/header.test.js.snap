// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render hesder correctly 1`] = `
<header>
  <h1>
    Spendee
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/Create"
  >
    Create Expense
  </NavLink>
  <button
    onClick={[Function]}
  >
    Logout
  </button>
</header>
`;
